pkg/runtime/wfengine/wfengine.go

```diff
@@ -168,7 +172,7 @@ func (wfe *WorkflowEngine) SetActorReminderInterval(interval time.Duration) {
 	wfe.activityActor.reminderInterval = interval
 }
 
-func (wfe *WorkflowEngine) Start(ctx context.Context) error {
+func (wfe *WorkflowEngine) Start(ctx context.Context) (err error) {
```

Comment:
What's the reason for this change?

Reply:
Oh it must have been something I did for some changes and then didn’t need it anymore. I’ll revert this

Reply:
Just so the `err` inside the loop doesn’t need to be instantiated in the loop. A bit like a small nit of mine :)